tests/testthat/helper-describe.R

```diff
@@ -0,0 +1,30 @@
+expect_describe_it_captures <-function(code_source, parent_desc, desc, captures) {
```

Comment:
Not entirely sure where to put this, but without such a helper, the tests were extremely long and repetitive. This is highly specific to the `test_that.scm` query, obviously.